modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1150,6 +1150,45 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Search for an object in the object list"
+	@priority = "5"
+	test SearchForObject {
+		property portal.acceptance = "true";
```

Comment:
Hey @nkkwon , could you please change it to "false"? I don't want to have it as an acceptance/relevant test case. Thanks!